recipes/iscan/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "iscan" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f1b209154df73c67743e53b203fe41c354a079cdd943018a2f84aa642f27dd8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - iscan=iscan.scan:main
```

Reply:
@marcelotrevisani Thanks for the suggestion! Do I still need to specify an entry point in the recipe if I already have one specified in my [setup.py](https://github.com/ZhengnanZhao/iscan/blob/master/setup.py#L26)? I'm inclined to say yes, since `setup.py` isn't included in PyPI tarball this recipe is sourcing from; therefore, conda-build isn't aware of the existence of `setup.py` and by extension this entry point, unless explicitly specified in the recipe, as you suggested. Is that correct?

Reply:
For console scripts you will need to specify those
Please see
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python